### PR TITLE
fix aspect permission test flapper

### PR DESCRIPTION
**Pull request by iamigo**

### CODE REVIEW

File: tests/api/v1/generators/post.js [hunk 1]
```diff
@@ -269,7 +269,7 @@ describe('tests/api/v1/generators/post.js >', () => {
       .catch((e) => done(e));
     });
 
-    it('error, all aspects do not have user permission', (done) => {
+    it.only('error, all aspects do not have user permission', (done) => {
```
Comment (jgraff2) [inline tests/api/v1/generators/post.js]: left in by mistake?

Comment (iamigo) [reply tests/api/v1/generators/post.js]: i hate when i do that :(

File: tests/api/v1/generators/post.js [hunk 2]
```diff
@@ -295,9 +295,9 @@ describe('tests/api/v1/generators/post.js >', () => {
       })
       .then((asp2) => { // created aspect2
         aspects.push(asp2.name);
-        asp2.addWriter(user2); // assign user2 as writer to aspect2
-        return validateGeneratorAspectsPermissions(aspects, req);
+        return asp2.addWriter(user2); // assign user2 as writer to aspect2
       })
+      .then(() => validateGeneratorAspectsPermissions(aspects, req))
```
Comment (jgraff2) [inline tests/api/v1/generators/post.js]: Good catch. Should it be the same on line 291?

Comment (iamigo) [reply tests/api/v1/generators/post.js]: changed a few more, yeah

Comment (iamigo) [reply tests/api/v1/generators/post.js]: tests passing

**Pull request merged**